be/src/Infrastructure/Localization/Foundation.Localization/Resources/GlobalResources.de.resx

```diff
@@ -166,7 +166,7 @@
     <value>Link kopieren</value>
   </data>
   <data name="Copyright" xml:space="preserve">
-    <value>Copyright © [year]</value>
+    <value>© [year] | ELTAP Spółka z ograniczoną odpowiedzialnością Sp.k. Alle Rechte vorbehalten.</value>
```

Comment:
**suggestion:** Hi @SebaKas21, there is 'ELTAP' hardcoded. The place of this text is either in the content API. In my view the easiest will be to revert the previous version